src/styles/_mixins.scss

```diff
@@ -353,6 +355,14 @@
   }
 }
 
+@mixin status-color($prop: 'color') {
```

Comment:
I saw this mixin somewhere, maybe in Polkaswap-web. Please check it and avoid duplications